src/simple_tsit5_integrator/tsit5_integrator.jl

```diff
@@ -38,45 +44,62 @@ function _build_caches(::MinimalTsit5, ::Type{T}) where {T}
-
-    #btilde1 = convert(T,-0.00178001105222577714)
-    #btilde2 = convert(T,-0.0008164344596567469)
-    #btilde3 = convert(T,0.007880878010261995)
-    #btilde4 = convert(T,-0.1447110071732629)
-    #btilde5 = convert(T,0.5823571654525552)
-    #btilde6 = convert(T,-0.45808210592918697)
-    #btilde7 = convert(T,0.015151515151515152)
-
-    # see here for interpolation
-    # https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/master/src/dense/interpolants.jl#L254-L270
-    # evalpoly is "something like"  ((((2+x)x+5)x+6)x+2)x+1
-
-    return cs, as
+        #=a21=# convert(T,0.161),
+        #=a31=# convert(T,-0.008480655492356989),
+        #=a32=# convert(T,0.335480655492357),
+        #=a41=# convert(T,2.8971530571054935),
+        #=a42=# convert(T,-6.359448489975075),
+        #=a43=# convert(T,4.3622954328695815),
+        #=a51=# convert(T,5.325864828439257),
+        #=a52=# convert(T,-11.748883564062828),
+        #=a53=# convert(T,7.4955393428898365),
+        #=a54=# convert(T,-0.09249506636175525),
+        #=a61=# convert(T,5.86145544294642),
+        #=a62=# convert(T,-12.92096931784711),
+        #=a63=# convert(T,8.159367898576159),
+        #=a64=# convert(T,-0.071584973281401),
+        #=a65=# convert(T,-0.028269050394068383),
+        #=a71=# convert(T,0.09646076681806523),
+        #=a72=# convert(T,0.01),
+        #=a73=# convert(T,0.4798896504144996),
+        #=a74=# convert(T,1.379008574103742),
+        #=a75=# convert(T,-3.290069515436081),
+        #=a76=# convert(T,2.324710524099774)
+    )
+
+    rs = SVector{22, T}(
+        #=r11=# convert(T,1.0),
+        #=r12=# convert(T,-2.763706197274826),
+        #=r13=# convert(T,2.9132554618219126),
+        #=r14=# convert(T,-1.0530884977290216),
+        #=r22=# convert(T,0.13169999999999998),
+        #=r23=# convert(T,-0.2234),
+        #=r24=# convert(T,0.1017),
+        #=r32=# convert(T,3.9302962368947516),
+        #=r33=# convert(T,-5.941033872131505),
+        #=r34=# convert(T,2.490627285651253),
+        #=r42=# convert(T,-12.411077166933676),
+        #=r43=# convert(T,30.33818863028232),
+        #=r44=# convert(T,-16.548102889244902),
+        #=r52=# convert(T,37.50931341651104),
+        #=r53=# convert(T,-88.1789048947664),
+        #=r54=# convert(T,47.37952196281928),
+        #=r62=# convert(T,-27.896526289197286),
+        #=r63=# convert(T,65.09189467479366),
+        #=r64=# convert(T,-34.87065786149661),
+        #=r72=# convert(T,1.5),
+        #=r73=# convert(T,-4),
+        #=r74=# convert(T,2.5),
+    )
+
+    return cs, as, rs
 end
 
+#######################################################################################
+# Stepping
+#######################################################################################
 # IIP version for vectors and matrices
-function DiffEqBase.step!(integ::MinimalTsit5Integrator{true, T, S}) where {T, S}
+function step!(integ::MinimalTsit5Integrator{true, T, S}) where {T, S}
```

Comment:
this should extend

Reply:
I do `import DiffEqBase: init, step!, isinplace`

Reply:
yeah I removed that because it's against our style guide to extend without reference.